test/orogen/test_logger.rb

```diff
@@ -17,40 +17,40 @@ def deploy_subject_syskit_model
         describe "log rotation" do
             it "increments index in log file name after rotation" do
                 task = syskit_stub_deploy_configure_and_start
-                task.orocos_task.property("overwrite_existing_files").write(false)
-                task.orocos_task.property("auto_timestamp_files").write(false)
+                task.properties.overwrite_existing_files = false
+                task.properties.auto_timestamp_files = false
```

Comment:
So.

The thing with the "stub" part is that it actually does not run the component. I.e. you don't need to set these here.

What you did forget is to forcefully set these parameters in the `update_properties` of the extension block.